presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -154,6 +155,11 @@ public SystemSessionProperties(
                             return concurrency;
                         },
                         value -> value),
+                integerSessionProperty(
+                        LOCAL_EXCHANGE_CONCURRENCY,
+                        "Number of local parallel hash partitioners per local exchange",
+                        1,
```

Comment:
I don't think we should add a new session property for this, because it will change the behavior of task concurrency of operators above the exchange that do not need partitioning.  For example
```sql
explain (type distributed)
select o.orderkey + 42 
from tpch.tiny.orders o join tpch.tiny.lineitem l on (o.custkey = l.linenumber);
```
The join in fragment 1 will not need `LocalExchange[ROUND_ROBIN]` on the probe side after this change, because the exchange operator is inherently a parallel operator with arbitrary partitioning.